modules/boot/src/utils/link.cpp

```diff
@@ -10,6 +10,8 @@
 #include "tools.h"
 #include "rels/include/defines.h"
 
+u32 t = 0;
+bool COUNT_STOP = 0;
```

Comment:
i'd probably give these some better names to make sure there's no confusion in the future, like `l_frameCount` instead of t, and `l_counterStopped` for COUNT_STOP

Reply:
can also make them static and put them inside the function, just for organization's sake. This is fine to do as long as its not used in any other function